### PR TITLE
add reaction methods to prototype

### PR DESCRIPTION
**Pull request by tyler-g**
Added the four reaction methods as specific by slack's RTM api: https://api.slack.com/methods

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -292,6 +292,72 @@ Bot.prototype.postTo = function(name, text, params, cb) {
...
+Bot.prototype.postReactionToChannel = function(id, emoji, ts, params) {
+    params = extend({
+        channel: id,
+        name: emoji,
+        timestamp: ts
+    }, params || {});
+
+    return this._api('reactions.add', params);
+};
+
```
Comment (alexellis) [inline index.js]: Does this work for you? I'm writing a bot to react to DMs and I keep getting `{ ok: false, error: 'no_item_specified' }` when calling this followed by .catch(function(e){  })

Comment (tyler-g) [inline index.js]: Hm.. are you passing the timestamp parameter?   Could you post some sample code?

**Comment by mitchcapper** [comment]
I tested postReactionToChannel / removeReactionFromChannel work perfectly to me :) Nice additions!

**Comment by mishk0** [comment]
The master has changed, could you please update your pr.

**Pull request closed without merging**